{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -1,24 +1,17 @@
 [tox]
-envlist = py34-coverage
+envlist = py34
 skipsdist = True
 
-[base]
-
 [testenv]
-setenv =
```

Comment:
Do you mean the envdir/ in general or specific to the tests?
Since the project is setup via envdir/, I changed the testing suite to use those because I wanted to have a single point of configuration... "don't repeat yourself" principle and stuff.